ruby_on_rails/sidekiq.md

```diff
@@ -0,0 +1,82 @@
+# Sidekiq
+
+* Add the following gem `gem 'sidekiq'` and install it
+
+```sh
+bundle install
+```
+
+* update the `Procfile` and set the `worker` line, so it looks like this:
+
+```rb
+web: bundle exec puma -C config/puma.rb
+worker: bundle exec sidekiq -C config/sidekiq.yml
+```
+
+* configure the active job adapter in `config/environments/production.rb`
+
+```rb
+config.active_job.queue_adapter = :sidekiq
+```
+
+* configure the active job adapter in `config/environments/development.rb`
+
+```rb
+config.active_job.queue_adapter = :inline
+```
+
+* configure the active job adapter in `config/environments/test.rb`
+
+```rb
+config.active_job.queue_adapter = :test
+```
+
+* create file `config/sidekiq.yml`
```

Comment:
create a file and specify that this is an example but may change depending on specific configuration